### ISSUE TITLE
v12: RADIUS requests are rejected if switch is not defined by IP address

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#7079]
**Describe the bug**
On Venom tests, RADIUS requests sent by Cumulus Switch are rejected. They look like this:

```
07:53:04.388309 IP (tos 0x0, ttl 64, id 30838, offset 0, flags [none], proto UDP (17), length 197)
    172.18.200.201.38877 > 172.18.200.10.1812: [bad udp cksum 0xe9bb -> 0x64ca!] RADIUS, length: 169
        Access-Request (1), id: 0x02, Authenticator: fe2e732f270ba90d3a2342e3694b98b0
          User-Name Attribute (1), length: 14, Value: 000300111101
            0x0000:  3030 3033 3030 3131 3131 3031
          User-Password Attribute (2), length: 18, Value:
            0x0000:  ccb2 70fd a19a 9849 488a 39af ce88 bd02
          NAS-Identifier Attribute (32), length: 11, Value: localhost
            0x0000:  6c6f 6361 6c68 6f73 74
          Called-Station-Id Attribute (30), length: 20, Value: 44-38-39-00-00-12:
            0x0000:  3434 2d33 382d 3339 2d30 302d 3030 2d31
            0x0010:  323a
          NAS-Port-Id Attribute (87), length: 7, Value: swp12
            0x0000:  7377 7031 32
          NAS-Port-Type Attribute (61), length: 6, Value: Ethernet
            0x0000:  0000 000f
          NAS-Port Attribute (5), length: 6, Value: 8
            0x0000:  0000 0008
          Service-Type Attribute (6), length: 6, Value: Call Check
            0x0000:  0000 000a
          Calling-Station-Id Attribute (31), length: 19, Value: 00-03-00-11-11-01
            0x0000:  3030 2d30 332d 3030 2d31 312d 3131 2d30
            0x0010:  31
          Acct-Session-Id Attribute (44), length: 18, Value: 4DF9145B72C2D2DE
            0x0000:  3444 4639 3134 3542 3732 4332 4432 4445
          Framed-MTU Attribute (12), length: 6, Value: 1400
            0x0000:  0000 0578
          Message-Authenticator Attribute (80), length: 18, Value: ......./.u...1.!
            0x0000:  8c1a 08fa 0ba3 8d2f a775 e9a0 a031 8021
```

=> There is no NAS-IP attribute
=> Called-Station-Id is the MAC address of `swp12` interface on Switch

**To Reproduce**
1. Create a switch in PacketFence **by IP**
3. Send a RADIUS request from this switch **without** `NAS-IP` attribute

=> You can see this message in `radius.log`: 

```
Aug  4 07:46:46 pfel8dev radiusd-auth-docker-wrapper[5971]: Thu Aug  4 07:46:46 2022 : Info: Adding client 100.64.0.1/32
Aug  4 07:46:46 pfel8dev radiusd-auth-docker-wrapper[5971]: Thu Aug  4 07:46:46 2022 : Info: Dropping packet without response because of error: Received packet from 100.64.0.1 with invalid Message-Authenticator!  (Shared secret is incorrect.) (from client 100.64.0.1/32)
```

**Expected behavior**
Request is handled by PacketFence because it looks at source IP address used to sent RADIUS request (the switch here)

**Additional context**
- If I created switch by MAC address, it works.
- I opened this issue because currently (on maintenance/11.{0,1,2}, having switch only defined by IP works. On top of that, I explicitely removed switch defined by MAC address in Venom tests last year. IIRC, it was related to a fix.
- RADIUS secret used to communicate with Cumulus switch is `testing123`
- As suggested by @fdurand, I duplicated `127.0.0.1` section in `raddb/clients.conf` and use `ipaddr=100.64.0.1`. Now I got this message in `radius.log`:
```
Aug  5 13:02:14 pfel8dev radiusd-auth-docker-wrapper[133781]: Fri Aug  5 13:02:14 2022 : ERROR: (0) rest: ERROR: Server returned:
Aug  5 13:02:14 pfel8dev radiusd-auth-docker-wrapper[133781]: Fri Aug  5 13:02:14 2022 : ERROR: (0) rest: ERROR: {"control:PacketFence-UserName":"000300111101","control:PacketFence-Mac":"00:03:00:11:11:01","control:PacketFence-Switch-Id":"100.64.0.1","control:PacketFence-Switch-Mac":"44:38:39:00:00:12","control:PacketFence-Authorization-Status":"allow","control:PacketFence-IfIndex":8,"control:PacketFence-Request-Time":1659704534,"Reply-Message":"Network device does not support this mode of operation","control:PacketFence-Connection-Type":"Ethernet-NoEAP","control:PacketFence-Switch-Ip-Address":"100.64.0.1","control:PacketFence-Eap-Type":0}

```

**Comment by nqb** [comment]
@fdurand, can we close this issue ?

EDIT: can we considered this issue as solved for v12.0.0 ? And keep it opened when radiusd-auth is containerized again ?

**Comment by nqb** [comment]
I will keep Venom tests with only switch declared by IP.
